deployments/ocean/image/binder/Dockerfile

```diff
@@ -1,2 +1,2 @@
 # Note that there must be a tag
-FROM pangeo/pangeo-notebook-onbuild:2019.07.01
+FROM pangeo/pangeo-notebook-onbuild:2019.08.08
```

Comment:
This update to ocean, which included numpy and xarray updates, broke several ongoing science projects.

We need to find a way to do these updates in a way that causes less friction for users.

Reply:
@rabernat - you are welcome to rollback this change. Ocean.pangeo.io was having a bunch of problems last week and we made a number of changes (image included) to attempt to fix the problem.